source/openapi-spec.yml

```diff
@@ -265,17 +307,22 @@ components:
           example: University of Huddersfield
         awarding_body_name:
           type: string
-          description: "Optional. The awarding body’s name, e.g. AQA"
+          description: "The awarding body’s name, e.g. AQA"
         awarding_body_country:
           type: string
-          description: The awarding body’s country as an ISO 3166 country code
+          description: "The awarding body’s country as an ISO 3166 country code"
           example: GB
         equivalency_details:
           type: string
-          description: Optional. Details of equivalency, if this qualification was awarded overseas
+          description: "Details of equivalency, if this qualification was awarded overseas"
     Reference:
       type: object
       additionalProperties: false
+      required:
+        - reference_type
+        - email
+        - relationship
+        - content
```

Comment:
What about `reason_for_character_reference` and `name`?

Reply:
Will we only return that field if it's a character reference? It's what the description says.